nsxt/resource_nsxt_policy_host_transport_node_profile.go

```diff
@@ -0,0 +1,161 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func resourceNsxtPolicyHostTransportNodeProfile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyHostTransportNodeProfileCreate,
+		Read:   resourceNsxtPolicyHostTransportNodeProfileRead,
+		Update: resourceNsxtPolicyHostTransportNodeProfileUpdate,
+		Delete: resourceNsxtPolicyHostTransportNodeProfileDelete,
+		Importer: &schema.ResourceImporter{
+			State: nsxtPolicyPathResourceImporter,
+		},
+		Schema: map[string]*schema.Schema{
+			"nsx_id":       getNsxIDSchema(),
+			"path":         getPathSchema(),
+			"display_name": getDisplayNameSchema(),
+			"description":  getDescriptionSchema(),
+			"revision":     getRevisionSchema(),
+			"tag":          getTagsSchema(),
+			// host_switch_spec
+			"standard_host_switch":      getStandardHostSwitchSchema(),
+			"preconfigured_host_switch": getPreconfiguredHostSwitchSchema(),
```

Comment:
looks like this attr is ignored later on?

Reply:
`getHostSwitchSpecFromSchema()` and `setHostSwitchSpecInSchema()` handle these two attributes (it's the same schema which is used by transport_node).